_posts/2020-06-20-Basic-Querying.md

```diff
@@ -0,0 +1,302 @@
+
+For introductory reading on the genealogies, we recommend Nollaig Ó Muraíle’s published lecture, _[The Irish Genealogies: Irish History’s Poor Relation?](https://www.ria.ie/its-occasional-lecture-vol-3-irish-genealogies-irish-historys-poor-relation)_, and Matthew Holmberg’s PhD thesis, ‘[Towards a Relative Chronology of the Milesian Genealogical Scheme](https://dash.harvard.edu/handle/1/37945002)’ (pp. 3–54). For an overview of IrishGen and our use of RDF, see the project [README](https://github.com/cyocum/irish-gen). All our data is available in our [GitHub repository](https://github.com/cyocum/irish-gen).
+
+## Sources
+The [Book of Leinster](https://www.vanhamel.nl/codecs/Dublin,_Trinity_College,_MS_1339) (**L**) and [Rawl. B.502](https://www.vanhamel.nl/codecs/Oxford,_Bodleian_Library,_MS_Rawlinson_B_502) (**R**) continue to be our main manuscript sources, as these contain the only genealogical collections currently in the IrishGen database that include significant quantities of material on the Cíannachta. The third major genealogical collection in IrishGen, that from [Laud Misc. 610](https://www.vanhamel.nl/codecs/Oxford,_Bodleian_Library,_MS_Laud_Misc._610) (**Ld**), also makes an appearance.
+
+A table of texts from the genealogies cited in this post is given at the end.
+
+## The Sons of Findchán
+We have seen, notwithstanding isolated variants, that the genealogies (Text 1) hold that the first descendant of Tadg mac Céin to acquire territory in Glengiven was Findchán, son of Fíac, son of Findchad Ulach and that he had eight sons. We can check the statement that he had eight sons against the rest of the IrishGen database. This can be done using a visual explorer, such as that provided by *[GraphDB](https://www.ontotext.com/try-graphdb-se/)*, which was used to produce the visualizations in the last blogpost. It can also be done using a [SPARQL](https://www.w3.org/TR/rdf-sparql-query/) query:
+
+```
+PREFIX rel: <http://purl.org/vocab/relationship/>
+SELECT ?s
+WHERE
+{
+    ?s rel:childOf <http://example.com/Rawl_B502/clanna_ébeir_h_i_l_leith_chuind.trig#Findchán>.
+}
+```
+
+This returns not just every child of Findchán given in **[R §1347](https://celt.ucc.ie/published/G105003/text018.html)** (Text 1), which is the mention of Findchán to which the URL used relates. It returns the URL of every child of someone predicated as the same as Findchán (`owl:sameAs`) across IrishGen and of everyone predicated as the same as each of those children, or predicated as the same as someone who is the same as Findchán or as one of his children and so on.
+
+25 URLs are returned:
+
+<img src="{{site.baseurl}}/assets/images/blogpost3_img#1.png" />
```

Comment:
This image is broken.  This is due to the fact that you used a '#' in the image name.  This has to be "url encoded" to work correctly (see https://en.wikipedia.org/wiki/Percent-encoding)  so this needs to be blogpost3_img%231.png to work (https://www.urlencoder.org/).

Reply:
I just removed the # instead